### ISSUE TITLE
Pkg3: TOML Compatibility

### ISSUE THREAD

**Issue by GravityAssisted** [JuliaLang/Juleps#35]
Hi,

I saw that you are using TOML. 

So Are you using TOML.jl package ? It seems that package is not maintained and doesn't support TOML V>0.2. 

thanks

**Comment by StefanKarpinski** [comment]
No, it's using a different, unregistered, TOML implementation that's linked to in the Julep. It will be kept up-to-date and will support whatever the current TOML spec is. We may have to take back the TOML.jl package name. Currently @wildart's TOML package is a bit unmaintained and @wildart himself is M.I.A., but if necessary, we can fork it and make the fork the official version.

**Comment by GravityAssisted** [comment]
@StefanKarpinski  Thanks, this is exciting. 
I am actually planing to use TOML as an input file parser in a tool we are developing here, in Julia. The input file has to be human readable but simple. So TOML looks like a great option. 
I was wondering how soon are you planning to replace the TOML.jl with your version ? or is it possible for us to to use your TOML now ?

**Comment by ararslan** [comment]
@GravityAssisted You can use the unregistered TOML parsing package by doing
```julia
Pkg.clone("https://github.com/wildart/TOML.jl.git")
```
But the usual cautions apply to using any package on its master branch: breaking changes could happen at any time, things might not work at any given time, etc. etc.

**Comment by GravityAssisted** [comment]
@ararslan Thanks I tried using it. Looks like it works fine for most of the cases, except "offset".  See issue [here](https://github.com/wildart/TOML.jl/issues/2)

this is okay, as I wont be using this feature I think.
